### PR TITLE
JMAP Standard Publication by IETF

### PR DESCRIPTION
**Pull request by yvesdaxmaz**
The specifications for JMAP Email have been published by the Internet Engineering Task Force (IETF)

### CODE REVIEW

File: news.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ permalink: /news.html
 title: JMAP News
 ---
 # JMAP News
+## JMAP Email Specification have been published by Internet Engineering Task Force (IETF), marking a big victory for email tools now and in the future
+
+16 August 2019
+
+>Fastmail engineers have been working for the past five years to bring [JMAP technology](https://fastmail.blog/2014/12/23/jmap-a-better-way-to-email/) to developers and customers. We're proud to announce that [RFC 8620](https://tools.ietf.org/html/rfc8620), published on July 18th, and [RFC 8621](https://tools.ietf.org/html/rfc8621), published on August 13th, have introduced JMAP to the world!
+
+[Continue reading...](https://fastmail.blog/2018/12/27/jmap-is-on-the-home-straight/)
```
Comment (neilj) [inline news.md]: This is not the correct URL; it should be https://fastmail.blog/2019/08/16/jmap-new-email-open-standard/

File: news.md [hunk 2]
```diff
@@ -4,6 +4,19 @@ permalink: /news.html
...
+
+## JMAP Protocol development funding
```
Comment (neilj) [inline news.md]: I'm not sure what this section is meant to be for; I think it can be cut.

**Review by neilj — CHANGES_REQUESTED** [review]

File: news.md (hunk 2 above)
Comment (yvesdaxmaz) [reply news.md]: I have remove this section

File: news.md (hunk 1 above)
Comment (yvesdaxmaz) [reply news.md]: I changed the link to point to the correct link

File: news.md (hunk 2 above)
Comment (marclaporte) [reply news.md]: The idea was to share this good news: https://github.com/k9mail/k-9/issues/3272#issuecomment-528326161

Comment (neilj) [reply news.md]: That sounds good if you want to add that in!

Comment (yvesdaxmaz) [reply news.md]: Should I restore that section?

Comment (neilj) [reply news.md]: I'm happy to have a section for https://github.com/k9mail/k-9/issues/3272#issuecomment-528326161 – this wasn't what was there before.

Comment (yvesdaxmaz) [reply news.md]: The section should concern [JMAP protocol issue #3272](https://github.com/k9mail/k-9/issues/3272l) or specificly the comment [k9mail/k-9#3272 (comment)](https://github.com/k9mail/k-9/issues/3272#issuecomment-528326161)

**Comment by marclaporte** [comment]
Let's close this, as it's superseded by the much better: https://github.com/jmapio/jmapio.github.io/pull/14/files

**Pull request closed without merging**